README.md

```diff
@@ -133,7 +133,7 @@ After getting familiar with the basic scenario, you may replace the test caniste
 2. Start a local replica instance:
     ```bash
    SNS_TESTING_INSTANCE=$(
-       docker run -p 8000:8000 -p 8080:8080 -v "`pwd`":/dapp -d ghcr.io/dfinity/sns-testing:main dfx start --clean
+       docker run -p 8000:8000 -p 8080:8080 -v "`pwd`":/dapp -d ghcr.io/dfinity/sns-testing:v1-legacy dfx start --clean
```

Comment:
Note to reviewer.

After this PR is merged, we'll tag the resulting commit as `v1-legacy`, which will make the instructions in the README file on that tag "frozen in time", documenting a way to run sns-testing in the legacy flow.

This will allow focusing these instructions exclusively on the new, 1-proposal flow from now on.